src/frontend/komponenter/Oppgaver/OppgaveList.tsx

```diff
@@ -19,77 +22,210 @@ const getEnheter = (enhetId: string) => {
-                    </tr>
-                </thead>
-                <tbody className="tabell__body">
-                    {oppgaver.status === RessursStatus.SUKSESS &&
-                        oppgaver.data.map((oppg: IOppgave, index) => (
-                            <tr key={index}>
-                                <td className={'regdato'}>
-                                    {intDatoTilNorskDato(oppg.opprettetTidspunkt)}
-                                </td>
-                                <td className={'oppgavetype'}>
-                                    {
-                                        OppgavetypeFilter[
-                                            oppg.oppgavetype as keyof typeof OppgavetypeFilter
-                                        ]
-                                    }
-                                </td>
-                                <td className={'gjelder'}>
-                                    {oppg.behandlingstema
-                                        ? GjelderFilter[
-                                              oppg.behandlingstema as keyof typeof GjelderFilter
-                                          ]
-                                        : 'Ikke satt'}
-                                </td>
-                                <td className={'frist'}>
-                                    {intDatoTilNorskDato(oppg.fristFerdigstillelse)}
-                                </td>
-                                <td className={'prioritet'}>
-                                    {
-                                        PrioritetFilter[
-                                            oppg.prioritet as keyof typeof PrioritetFilter
-                                        ]
-                                    }
-                                </td>
-                                <td>
-                                    <div className={'beskrivelse'}>{oppg.beskrivelse}</div>
-                                </td>
-                                <td className={'bruker'}>{oppg.aktoerId}</td>
-                                <td className={'enhet'}>{getEnheter(oppg.tildeltEnhetsnr)}</td>
-                                <td className={'saksbehandler'}>
-                                    {oppg.tilordnetRessurs ? oppg.tilordnetRessurs : 'Ikke tildelt'}
-                                </td>
-                                <td className={'handlinger'}>
-                                    {OppgavetypeFilter[
-                                        oppg.oppgavetype as keyof typeof OppgavetypeFilter
-                                    ] === OppgavetypeFilter.JFR && (
-                                        <a href={`/oppgaver/journalfør/${oppg.id}`}>Gå til oppg</a>
-                                    )}
-                                </td>
+        <div>
+            <div className={'oppgavelistHeader'}>
+                <Systemtittel className={'oppgavelistHeader__tittel'}>
```

Comment:
```suggestion
            <div className={'oppgavelist-header'}>
                <Systemtittel className={'oppgavelist-header__tittel'}>
```

We try to follow BEM standard for css, so no camelcase :)